MixItUp.Desktop/Services/DesktopTelemetryService.cs

```diff
@@ -54,5 +56,18 @@ public void End()
             this.telemetryClient.Flush();
             Task.Delay(1000); // Allow time to flush
         }
+
+        private void RefreshUserDetails()
+        {
+            if (string.IsNullOrEmpty(this.telemetryClient.Context.User.Id))
+            {
+                if (string.IsNullOrEmpty(ChannelSession.Settings.TelemetryUserId))
+                {
+                    ChannelSession.Settings.TelemetryUserId = Guid.NewGuid().ToString();
```

Comment:
Why not do this somewhere here?

https://github.com/SaviorXTanren/mixer-mixitup/blob/master/MixItUp.Desktop/DesktopChannelSettings.cs#L373

Reply:
Good call, fixing.